src/main/java/frc/robot/commands/Arm/ScoreInAmp.java

```diff
@@ -21,6 +21,8 @@ public ScoreInAmp(PizzaBoxSubsystem pizzaBoxSubsystem, ArmSubsystem armSubsystem
         new StopNoteContainerMotor(pizzaBoxSubsystem),
         new SpinToArmAngle(armSubsystem, ArmSubsystem.Arm.INTAKE_ANGLE)
     );
+
+    pizzaBoxSubsystem.hasNote = false;
```

Comment:
Should this be in initialize, or the first 'InstantCommand' so it gets set false every time?